clifford/jit_func.py

```diff
@@ -0,0 +1,80 @@
+
+import ast
+try:
+    import astpretty
+    AST_PRETTY_AVAILABLE = True
+except:
+    AST_PRETTY_AVAILABLE = False
+import inspect
+import warnings
+from ._numba_utils import njit
+from ._ast_transformer import GATransformer
+
+
+class jit_func(object):
+    """
+    This is a JIT decorator that re-writes the AST and then numba JITs
+    the resulting function.
+    """
+    def __init__(self, layout, ast_debug=False):
+        self.layout = layout
+        if AST_PRETTY_AVAILABLE:
+            self.ast_debug = ast_debug
+        else:
+            if ast_debug:
+                warnings.warn('''
+The ast_debug flag is set to True, but the astpretty module is not importable.
+To see ast_debug output please pip install astpretty
+''')
```

Comment:
```suggestion
                warnings.warn(
                    'The ast_debug flag is set to True, but the astpretty module is not importable.\n'
                    'To see ast_debug output please pip install astpretty')
```
or `textwrap.dedent`, but that seems overkill for two lines of text.